source/projects/private-voice-assistant/voice_remote_classic_phone.markdown

```diff
@@ -0,0 +1,154 @@
+---
+title: "Setting up an analog phone to control Home Assistant"
+---
+
+This tutorial shows how to set up an analog phone to control Home Assistant with your voice.
+The phone is used to implement a push-to-talk mechanism. 
+In this way, the beginning and end of a voice stream are clearly defined.
+This reduces the complexity of voice processing. 
+
+It also means that Home Assistant only listens when you want it to.
+
+<lite-youtube videoid="bPDyDFUHDCk" videotitle="Using an analog phone to control Home Assistant"></lite-youtube>
+
+## Required material
+
+* Latest version of Home Assistant installed on your Home Assistant device
+* [Analog phone](#about-the-analog-phone)
+* Analog telephone adapter 
+  [Grandstream HT801](https://www.grandstream.com/products/gateways-and-atas/analog-telephone-adaptors/product/ht801) or 
+  [HT802](https://www.grandstream.com/products/gateways-and-atas/analog-telephone-adaptors/product/ht802)
+  * includes a 5&nbsp;V power adapter and an Ethernet cable
+* RJ11 cable to connect the phone to the Grandstream
+
+## Setting up Grandstream
+
+1. Connect the RJ11 cable from the phone to the green socket on the Grandstream.
+   ![Grandstream HT801 ports](/images/assist/grandstream-ht801-interfaces.png)
+1. Connect the Grandstream to your network: 
+   * Plug the Ethernet cable into the blue socket and connect it to your router.
+1. Start up the Grandstream.
+   * Connect the power supply.
+   * Once the Grandstream has booted, the two LEDs for power and Ethernet light up solid blue. The phone LED won't light up.
+1. Log onto your router and identify the IP address of the Grandstream.
```

Comment:
You can also get the phone to tell you its IP address! https://documentation.grandstream.com/knowledge-base/ht801-quick-installation-guide/#obtaining-the-ip-address

Reply:
That's cool!